pyxodr/road_objects/road.py

```diff
@@ -552,6 +576,35 @@ def junction_connecting_ids(self) -> Dict[str, Set[int]]:
 
         return _junction_connecting_ids
 
+    @cached_property
+    def road_properties(self) -> RoadProperties:
+        """Return all the existing road properties for this road."""
+        length = float(self["length"])
+        name = self["name"]
+        type_element = self["type"]
+
+        max_speed = None
+        if type_element is not None:
+            speed_element = type_element["speed"]
+            if speed_element is not None:
+                try:
+                    max_speed = float(speed_element["max"])
+                    speed_unit = speed_element.get("unit", "mps")
+                    if speed_unit == "mph":
+                        max_speed = max_speed * 0.44704
+                    elif speed_unit == "km/h":
+                        max_speed = max_speed * 0.277778
+                    else:
+                        max_speed = max_speed
+                except Exception:
+                    print(f"Couldn't convert speed in {speed_unit}")
+        else:
+            print("No <type> element found.")
```

Comment:
We don't need to add console output to flag that data that's not required is not contained in the input file.